modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/OrderableTable.tsx

```diff
@@ -124,12 +183,22 @@ const Row = ({
 			ref={tableRowRef}
 		>
 			<ClayTable.Cell className="drag-handle-cell">
+				{tableRowRef?.current?.classList.contains('dragging') ? (
+					<span aria-live="assertive" className="sr-only">
+						{Liferay.Language.get(
+							'use-up-and-down-arrows-to-move-the-field-and-press-enter-to-place-it-in-desired-position'
+						)}
+					</span>
+				) : null}
+
 				<ClayButtonWithIcon
 					aria-label={Liferay.Util.sub(
 						Liferay.Language.get('drag-x'),
 						item.label || Liferay.Language.get('item')
 					)}
 					displayType={null}
+					onBlur={() => onBlur()}
+					onKeyDown={(event) => onKeyDown(event)}
```

Comment:
You can just do this
```suggestion
					onBlur={onBlur}
					onKeyDown={onKeyDown}
```

Reply:
Made the changes!